package/network/utils/umbim/files/lib/netifd/proto/mbim.sh

```diff
@@ -17,6 +17,7 @@ proto_mbim_init_config() {
 	proto_config_add_string auth
 	proto_config_add_string username
 	proto_config_add_string password
+	[ -e /proc/sys/net/ipv6 ] && proto_config_add_string ipv6
```

Comment:
I don't see any reason to check for system support here. Just honor whatever setting the user configured, as I did in https://github.com/TDT-AG/openwrt/pull/1/commits/76c27c2c0eab6f34e633517ca2a76ef916a2d67d

Reply:
I disagree. There is no need to do any of ipv6-related stuff if it isn't built into the kernel. Same check is done in QMI protocol handler.

Reply:
OK, fair enough